specification/TileFormats/FeatureTable/README.md

```diff
@@ -12,7 +12,7 @@
 
 ## Overview
 
-A _Feature Table_ describes position and appearance properties for each feature in a tile.  The [Batch Table](../BatchTable/README.md), on the other hand, contains per-feature application-specific metadata not necessarily used for rendering.
+A _Feature Table_ is a component of a tile's binary body and describes position and appearance properties required to render each feature in a tile. The [Batch Table](../BatchTable/README.md), on the other hand, contains per-feature application-specific metadata properties that a client uses to evaluate the declarative styling for a feature and may choose to use for further rendering or analysis.
```

Comment:
The batch table counter-summary seems too detailed. The original wording was easier to follow. The change from metadata to properties is still good to keep though.